static/css/styles.css

```diff
@@ -0,0 +1,7 @@
+body {
+    margin-top: 10px;
+}
+
+.sidebar-nav {
+    list-style: none;
+}
```

Comment:
Feel free to add more styles, also bootstrap to your template, it's good to have such skills, because when you work with clear Django ussualy you don't have a frontend developer who will make front with django templates)